Makefile

```diff
@@ -264,6 +264,7 @@ zip: build-lambdas ## Generate zips for deploying to AWS (src/js/grapl-cdk/zips/
 zip-pants: ## Generate Lambda zip artifacts using pants
 	./pants filter --filter-target-type=python_awslambda :: | xargs ./pants package
 	cp ./dist/src.python.provisioner.src/lambda.zip ./src/js/grapl-cdk/zips/provisioner-$(or $(TAG),latest).zip
+	cp ./dist/src.python.engagement-creator/engagement-creator.zip ./src/js/grapl-cdk/zips/engagement-creator-$(or $(TAG),latest).zip
```

Comment:
I hadn't noticed the previous use of the default value on the line above, but doing so is redundant, as we set TAG=latest by default: https://github.com/grapl-security/grapl/blob/main/Makefile#L6

Reply:
Good call; I can clean that up.